index.html

```diff
@@ -78,5 +80,221 @@ <h2 class="secondary-header">LET’S CONNECT</h2>
         </ul>
       </div>
     </section>
+    <!-- WORKS SECTION  -->
+    <section class="works">
+      <!-- first project  -->
+      <div class="projects container">
+        <img class="project-img" src="images/works/twitter-shot.jpg" alt="" />
+        <h3 class="tertiary-heading">Tonic</h3>
+        <ul class="project-details">
+          <li>CONOPY</li>
+          <li>Back End Dev</li>
+          <li>2015</li>
+        </ul>
+        <p class="project-description">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="project-lang">
+          <li>HTML</li>
+          <li>CSS</li>
+          <li>JavaScript</li>
+        </ul>
+        <button class="project-btn">See project</button>
+      </div>
```

Comment:
- [ ] Well done :clap: you did a very nice job :+1: but you are not using all button effects like `hover, active, pressed` Kindly add all button effects and please stick to the Figma design as well :slightly_smiling_face: 

![Screenshot from 2023-02-17 17-31-04](https://user-images.githubusercontent.com/45226848/234981664-e0742545-e691-4577-8ed6-921f9f2aea4e.png)